Source/WebCore/page/scrolling/AsyncScrollingCoordinator.h

```diff
@@ -160,14 +160,21 @@ class AsyncScrollingCoordinator : public ScrollingCoordinator {
     
     void applyScrollPositionUpdate(ScrollUpdate&&, ScrollType);
     void updateScrollPositionAfterAsyncScroll(ScrollingNodeID, const FloatPoint&, std::optional<FloatPoint> layoutViewportOrigin, ScrollingLayerPositionAction, ScrollType);
+    void animatedScrollWillStartForNode(ScrollingNodeID);
     void animatedScrollDidEndForNode(ScrollingNodeID);
+    void wheelEventScrollWillStartForNode(ScrollingNodeID);
+    void wheelEventScrollDidEndForNode(ScrollingNodeID);
 
     FrameView* frameViewForScrollingNode(ScrollingNodeID) const;
 
+    void hysterisisTimerFired();
+
     std::unique_ptr<ScrollingStateTree> m_scrollingStateTree;
     RefPtr<ScrollingTree> m_scrollingTree;
 
     bool m_eventTrackingRegionsDirty { false };
+
+    Timer m_scrollingHysterisisTimer;
```

Comment:
Can this use HysteresisActivity?

Reply:
Probably! I had no idea this class existed haha